barman/command_wrappers.py

```diff
@@ -1014,6 +1017,58 @@ def __init__(
             self.args += args
 
 
+class PgVerifyBackup(PostgreSQLClient):
+    """
+    Wrapper class for the pg_verify system command
+    todo: findout why script path is:
+    /usr/lib/postgresql/13/bin/
+    and not /usr/bin/
+    """
+
+    # Todo  find a clean way to find exec
+    COMMAND_ALTERNATIVES = [
+        "pg_verifybackup",
+        "/usr/lib/postgresql/13/bin/pg_verifybackup",
```

Comment:
I wish, but for some reason it is not always. I barman-testing docker images for example it is not in the PATH. I am not sure what is the best way to handle it.
Probably something to do at installation time. For docker images  do some fix in docker images to behave like exec is in the path 
`ENV PATH=$PATH:/usr/lib/postgresql/$PG_MAJOR/bin`
Hoping there will be no side effect with other pg binary we usually find in `usr/bin/`

What is your view on this?

Reply:
You're right - `postgresql-client-13` provides the `pg_verifybackup` command in `/usr/lib/postgresql/$PG_MAJOR/bin` (along with various other client applications) but it's the `postgresql-client-common` package which adds the `pg_*` commands which Barman uses into `/usr/bin` and `postgresql-client-common` does not include `pg_verifybackup`.

This feels like something which should be fixed at the packaging level, i.e. the barman OS package would make sure pg_verifybackup is in the path (and anyone installing from source would need to make sure it is in the path themselves).

Reply:
I have added a temporary fix adding pg_verifybackup path to PATH.
Not sure how to fix this at packaging level. I will check the repo.

Reply:
For the benefit of the PR, we discussed this synchronously and concluded we'll leave it up to the user to make sure the right version of the `pg_verifybackup` binary is on the path.

We'll update the installation docs to cover this (and also `pg_receivewal` since that has a similar issue) but that does not need to block this PR.